### ISSUE TITLE
NVM directory already exists

### ISSUE THREAD

**Issue by kceb** [dcodeIO/setup-node-nvm#3]
Getting the following error on macOS but not ubuntu:

```
fatal: destination path '/Users/runner/.nvm' already exists and is not an empty directory.
```

I can rm -rf that directory before the iOS step, but was wondering why it is happening suddenly. To my knowledge, we didn't cache that directory.

**Comment by kceb** [comment]
Looks like github added nvm to it's macOS containers. 

I think to resolve this, this package could check to see if an nvm install already exists.

**Comment by ljharb** [comment]
This would probably work fine if it only set `$NVM_DIR` if it wasn't already set?

**Comment by dcodeIO** [comment]
A while ago I changed the install directory from $HOME/.nvm to $HOME/mynvm ([see](https://github.com/dcodeIO/setup-node-nvm/blob/3bc09a3f6f58358a453eccc31a88b0c661cba9f9/install.sh#L3)). Looks a bit silly, but should work around the problem by replacing any potentially existing nvm install for the course of the workflow run.

**Comment by ljharb** [comment]
Right, but i think the ideal/goal is not to install an extra nvm/node at all - ie, to reuse the existing one.
